docs/cosmos-hub/upgrade-node.md

```diff
@@ -0,0 +1,99 @@
+# Upgrade Your Node
+
+::: warning
+The detailed procedure to upgrade a mainnet node from `cosmoshub-1` to `cosmoshub-2` can be found [here](https://gist.github.com/alexanderbez/5e87886221eb304b9e85ad4b167c99c8). 
```

Comment:
This is kinda incorrect. That is the procedure to generate the `genesis.json` file for `cosmoshub-2` from the `cosmoshub-1` state-dump at block `500,000`. I don't think we need to link to those here.